modules/custom_layers/README.md

```diff
@@ -0,0 +1,46 @@
+Module with guides how to enable in Intel OpenVINO some layers from PyTorch:
+
+* [torch.fft](examples/fft)
+* [nn.functional.grid_sample](/examples/grid_sample)
+
+And other custom operations:
+
+* [calculate_grid](/examples/calculate_grid) and [sparse_conv](/examples/sparse_conv) from [Open3D](https://github.com/isl-org/Open3D)
+* [complex_mul](/examples/complex_mul) from [DIRECT](https://github.com/NKI-AI/direct)
+
+
+## OpenVINO custom CPU extensions
+
+You  need to build CPU extensions library which actually has C++ layers implementations:
```

Comment:
```suggestion
## Build custom OpenVINO operation extension library

The C++ code implementing the custom operation is in the `user_ie_extensions` directory. You'll have to build an "extension library" from this code so that it can be loaded at runtime. The steps below describe the build process:
```